From 8d2e849a1b78fee6258f654f0a2fc2c48ef9c80a Mon Sep 17 00:00:00 2001
From: Hans Petter Selasky <hps@selasky.org>
Date: Thu, 6 Oct 2016 18:54:36 +0200
Subject: [PATCH 1046/1936] Build fix for FreeBSD.

major() and minor() are defined as macros by sys/types.h
and needs to be undefined to build Zynaddsubfx under
FreeBSD.

Signed-off-by: Hans Petter Selasky <hps@selasky.org>
---
 src/zyn-version.h.in | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/src/zyn-version.h.in b/src/zyn-version.h.in
index e5167eca..1222b150 100644
--- a/src/zyn-version.h.in
+++ b/src/zyn-version.h.in
@@ -17,6 +17,14 @@
 
 #include <iosfwd>
 
+#ifdef major
+#undef major
+#endif
+
+#ifdef minor
+#undef minor
+#endif
+
 //! class containing a zynaddsubfx version
 class version_type
 {
-- 
2.47.0

